src/style/app.module.css

```diff
@@ -520,18 +533,60 @@
   border: none;
 }
 
+.actionsButton {
+  background-color: var(--search-button-bg);
+  color: var(--brown-color);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`